.luacheckrc

```diff
@@ -155,23 +174,50 @@ globals = {
   "abstract_ferns", -- from ferns
   "abstract_youngtrees", -- from youngtrees
   "music", -- from music_api
+  "skins", -- from skinsdb
   table.unpack(mod_names)
 }
 
+exclude_files = {
+  -- Assets
+  "mods/livingjungle/schematics",
+  -- Legacy stuff
+  "mods/animalia/api/legacy_convert.lua",
+  "mods/minetest_game/default/legacy.lua",
+}
+
 -- Mod-specific configuration
 -----------------------------
 
 -- Import existing .luacheckrc files
+-- List generated with:
+--   find mods -name .luacheckrc -exec dirname {} \; | xargs -L1 basename | awk '{ sub("\r",""); print "  \"" $1 "\"," }' | sort
 local mods_with_luacheckrc = {
+  "animalia",
+  "awards",
   "creatura",
+  "everness",
   "flowerpot",
+  "item_drop",
+  "plantlife_modpack",
   "player_monoids",
+  "signs_lib",
+  "skinsdb",
   "stamina",
+  "telemosaic",
+  "x_farming",
 }
 for _,modname in ipairs(mods_with_luacheckrc) do
-  files["mods/"..modname] = {}
+  local modpath = "mods/" .. modname
+  files[modpath] = {}
   -- store globals defined in the mod's .luacheckrc in the corresponding files table
-  assert(pcall(assert(loadfile("mods/"..modname.."/.luacheckrc", nil, files["mods/"..modname]))))
+  assert(pcall(assert(loadfile(modpath.."/.luacheckrc", nil, files[modpath]))))
+  if files[modpath].exclude_files then
+    for i,path in ipairs(files[modpath].exclude_files) do
+      -- exclude_files only works in the global scope, not in files[]
+      table.insert(exclude_files, modpath.."/"..path)
+    end
+  end
 end
```

Comment:
If you want to avoid this complexity, another option is to completely ignore mods with .luacheckrc and check them in their own repos.